static/js/crowdsource.routes.js

```diff
@@ -40,6 +40,11 @@
     templateUrl: '/static/templates/tasksearches/tasklistSearch.html',
     controller: 'taskSearchGridController'
     })
+    .when('/work', {
```

Comment:
no need for a new route, we will use the same as the one we are using now to submit work for micro